# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1234941921 -3600
#      Wed Feb 18 08:25:21 2009 +0100
# Branch release-3-0-x
# Node ID 7518126401ac323c48921e22aeb8a110418950db
# Parent  205f10f5cbe85e4a97abd0f11982295c5d6c5f3f
reverse effect of 61951dcb846c

diff --git a/scripts/plot/contourf.m b/scripts/plot/contourf.m
--- a/scripts/plot/contourf.m
+++ b/scripts/plot/contourf.m
@@ -163,17 +163,17 @@ function varargout = contourf (varargin)
     cont_len(del_idx) = cont_idx(del_idx) = [];
   endif
 
   ## Now we have everything together and can start plotting the patches
   ## beginning with largest area.
   [tmp, svec] = sort (cont_area);
   len = ncont - numel (del_idx);
   h = zeros (1, len);
-  for n = 1:len
+  for n = len:-1:1
     idx = svec(n);
     ii = cont_idx(idx):cont_idx(idx) + cont_len(idx) - 2;
     h(n) = patch (c(1, ii), c(2, ii), cont_lev(idx), patch_props{:});
   endfor
 
   if (min (lev) == max (lev))
     set (gca (), "clim", [min(lev)-1, max(lev)+1]);
   else
